ibis/tests/all/test_join.py

```diff
@@ -10,7 +10,7 @@ def left(batting):
 
 @pytest.fixture(scope='module')
 def right(awards_players):
-    return awards_players[awards_players.lgID == 'NL']
+    return awards_players[awards_players.lgID == 'NL'].drop(['yearID', 'lgID'])
```

Comment:
Why are 'yearID' and 'lgID' dropped here?

Reply:
Having duplicate column names in the result confuses certain backends (for example PySpark). Fro the purpose of this test I think avoiding duplicate column names make our lives easier.